packages/boxel/addon/components/boxel/action-chin/action-status-area/index.gts

```diff
@@ -0,0 +1,24 @@
+import { svgJar } from '@cardstack/boxel/utils/svg-jar';
+import type { TemplateOnlyComponent } from '@ember/component/template-only';
+
+interface Signature {
+  Element: HTMLDivElement;
+  Args: {
+    icon: string;
+  };
+  Blocks: {
+    'default': [],
+  }
+}
+
+const StatusArea: TemplateOnlyComponent<Signature> = <template>
+  <div class="boxel-action-chin__action-status-area" ...attributes data-test-boxel-action-chin-action-status-area>
+    {{#if @icon}}
+      {{svgJar @icon class="boxel-action-chin__action-status-area-icon" width="20" height="20" role="presentation"}}
+    {{/if}}
+    {{yield}}
+  </div>
+</template>
+
+export default StatusArea;
```

Comment:
At the moment, if you want to put a correct signature on a template-only component you can't inline it into the `export default`. You need to assign it to a variable such that you can give the variable the type.